server/eb/README.md

```diff
@@ -108,29 +108,35 @@ To include these scripts in the deployment, use the following steps:
    $ grep inline_scripts config.yaml
      inline_scripts : [ myscript.js ]
    ```
+
+1. Optionally, you can add plugins to the server python code. To include a plugin in the deployment use the following steps:
```

Comment:
I have a PR out right now that adds custom requirements.   If your plugin pulls in new python modules, you can
place them in the custom requirements.txt file.   When both of these get to master, the docs should add a line for that.